vars/suse-15.yml

```diff
@@ -14,5 +14,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+_zypper_repository:
+  - name: "packman-x86_64"
+    repo: "http://packman.inode.at/suse/openSUSE_Leap_15.2/"
```

Comment:
OMG packman have `packman.repo` for Tumbleweed:
- http://packman.inode.at/suse/openSUSE_Tumbleweed/packman.repo